README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
 
-Please note that impress.js was created for developers. Some basic knowledge of html, css and
-javascript is a prerequisite to install and use its features.
+Please note that impress.js was created for developers. Some basic knowledge of HTML, CSS and
```

Comment:
I think these two paragraphs about being 'developer' and looking into examples or editing tools are the ones that make the whole documentation message sound bad for those that may not found themselves experienced.

So I would suggest removing them and exchanging with a link to a API documentation.